airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/concurrent/cursor.py

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, Iterable, MutableMapping
+
+from airbyte_cdk.sources.streams.concurrent.cursor import Cursor
+from airbyte_cdk.sources.streams.concurrent.partitions.partition import Partition
+from airbyte_cdk.sources.streams.concurrent.partitions.record import Record
+
+
+class FileBasedNoopCursor(Cursor):
```

Comment:
Should we avoid defining `state`, `observe` and `close_partition` as they are already part of the `Cursor` interface?